lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -42,6 +42,7 @@
 
 #include "SPIRVBasicBlock.h"
 #include "SPIRVEnum.h"
+#include "SPIRVFunction.h"
```

Comment:
I don't know why my change caused clang tidy to fail, but we need this include because of https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/11f2f38a829eb57f95ccb7558a8da5ab43c9462c/lib/SPIRV/libSPIRV/SPIRVInstruction.h#L1694 which clang tidy says should have defn of cast target